package.json

```diff
@@ -133,6 +133,7 @@
     "bundle-loader": "^0.5.4",
     "cbioportal-frontend-commons": "^0.2.7",
     "cbioportal-ts-api-client": "^0.9.0-beta.0",
+    "cbioportal-utils": "^0.1.0-beta.0",
```

Comment:
Not sure if this is the best name for the new package. Name suggestions are welcome!

Reply:
Since we have cbioportal-frontend-commons, maybe cbioportal-frontend-utils?

Reply:
I was thinking that too, but this package is intended to be used by node scripts/backend, too. So, "frontend" in the package name might be misleading.